_data/unternehmen_2023.json

```diff
@@ -96,6 +96,22 @@
             "Know-how in Data Science oder Softwareentwicklung"
         ]
     },
+    {
+        "name": "RAITH Nanofabrictions",
```

Comment:
Fabrication?

Reply:
Trueee (not visible anyway, only in url)